src/shared/components/adminTabs/Games/GamesManagement.tsx

```diff
@@ -53,11 +53,40 @@ function GamesManagement() {
         })])
     }
 
+    function areDistinct(arr)
+    {
+        let n = arr.length;
```

Comment:
Let's add a data type for the array in the parameter.
The code should look something like `function areDistinct(arr: NormalGame[])`